.github/workflows/release.yaml

```diff
@@ -0,0 +1,28 @@
+---
+name: Release crate
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "version (no `v` prefix)"
+        required: true
+        default: "1.0.0"
+
+jobs:
+  publish:
+    name: Publish Limitador crate to crates.io
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: crate-v${{ github.event.inputs.version }}
+      - name: Build
+        run: cargo build --verbose --release
+      - name: Check
+        run: cargo fmt --all -- --check && cargo clippy -- -D warnings
+      - name: Run tests
+        run: cargo test --verbose
+      - name: Publish
+        if: ${{ success() }}
```

Comment:
this is really needed? shouldn't the workflow end if a previous step fails?

Reply:
I wasn't sure tbh

Reply:
I think you are right and this can be deleted, I'll do it when I get rid of the `--dry-run`